lib/plugins/webpagetest/index.js

```diff
@@ -13,18 +13,39 @@ const fs = require('fs');
 // These are the metrics we want to save in
 // the time series database per pageSummary
 const DEFAULT_PAGE_SUMMARY_METRICS = [
-  'data.median.*.SpeedIndex',
-  'data.median.*.render',
-  'data.median.*.TTFB',
-  'data.median.*.loadTime',
-  'data.median.*.fullyLoaded',
-  'data.median.*.userTimes.*',
+  'data.median.firstView.SpeedIndex',
+  'data.median.firstView.render',
+  'data.median.firstView.TTFB',
+  'data.median.firstView.loadTime',
+  'data.median.firstView.fullyLoaded',
+  'data.median.firstView.userTimes.*',
   // Use bytesIn to collect data for Opera Mini & UC Mini
-  'data.median.*.bytesIn',
-  'data.median.*.breakdown.*.requests',
-  'data.median.*.breakdown.*.bytes',
-  'data.median.*.requestsFull'
+  'data.median.firstView.bytesIn',
+  'data.median.firstView.breakdown.*.requests',
+  'data.median.firstView.breakdown.*.bytes',
+  'data.median.firstView.requestsFull',
+
+  'data.median.firstView.custom.*',
+  'data.median.firstView.domContentLoadedEventEnd',
+  'data.median.firstView.fullyLoadedCPUms',
+  'data.median.firstView.docCPUms',
+  'data.median.firstView.score_cache',
+  'data.median.firstView.score_gzip',
+  'data.median.firstView.score_combine',
+  'data.median.firstView.score_minify',
+  'data.median.firstView.domElements',
+  'data.median.firstView.lastVisualChange',
+  'data.median.firstView.visualComplete85',
+  'data.median.firstView.visualComplete90',
+  'data.median.firstView.visualComplete95',
+  'data.median.firstView.visualComplete99',
+  'data.median.firstView.FirstInteractive',
+  'data.median.firstView.TimeToInteractive',
```

Comment:
does this prevent the data from repeat view to be processed? because it explicitly says `data.media.firstView...` now, instead of `data.median.*...`

Reply:
ooops, you are right, we changed the default behavior there, let me fix that. Usually I only test first view but we shouldn't change how we collect metrics.